app/assets/javascripts/app/discussions/show.html.slim

```diff
@@ -35,7 +35,8 @@
               a ng-click="ctrl.toogleDiscussionState(ctrl.discussion)" reopen discussion
 
       md-content.md-padding md-theme="white"
-        div comments-list="ctrl.discussion.comments" comment-count="ctrl.discussion.comments_count" toggle="ctrl.openEditForm(comment)"
+        div layout="column" comments-list="ctrl.discussion.comments" comment-count="ctrl.discussion.comments_count" toggle="ctrl.openEditForm(comment)"
+        comment-form ng-hide="ctrl.device.isMobile()" toggle="ctrl.toggleCommentForm()" mobile="ctrl.device.isMobile()" comment="ctrl.comment" discussion="ctrl.discussion"
 
-sliding-screen show="ctrl.showNewComment" hide-bottom=""
-  comment-form toggle="ctrl.toggleCommentForm()" discussion="ctrl.discussion" comment="ctrl.comment"
+sliding-screen show="ctrl.device.isMobile()" hide-bottom=""
```

Comment:
Yes, but now that you are pointing this I think I will create another template or another directive for the desktop form